vscode-lean4/src/config.ts

```diff
@@ -200,6 +200,33 @@ export function isElanDisabled() : boolean {
     return typeof(process.env.DISABLE_ELAN) === 'string';
 }
 
+/** The editor line height, in pixels. */
+export function getEditorLineHeight(): number {
+    // The implementation
+    // (recommended by Microsoft: https://github.com/microsoft/vscode/issues/125341#issuecomment-854812591)
+    // is absolutely cursed. It's just to copy whatever VSCode does internally.
+    const fontSize = workspace.getConfiguration('editor').get<number>('fontSize') ?? 0;
+    let lineHeight = workspace.getConfiguration('editor').get<number>('lineHeight') ?? 0;
+
+    const GOLDEN_LINE_HEIGHT_RATIO = process.platform === 'darwin' ? 1.5 : 1.35;
+    const MINIMUM_LINE_HEIGHT = 8;
+
+    if (lineHeight === 0) {
+		lineHeight = GOLDEN_LINE_HEIGHT_RATIO * fontSize;
```

Comment:
what's with the mix of spaces and tabs in our source files anyway, can we choose one (I prefer 4 spaces myself)...

Reply:
Yes, it's a mess (I also prefer 4 spaces). It is easy to miss these kinds of changes, we should really implement #38 .